drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -73,14 +73,14 @@ void increment_multi_counter(void)
 void test_multi_ticker(void)
 {
     LowPowerTicker ticker[TICKER_COUNT];
-    const uint32_t extra_wait = 10; // extra 10ms wait time
+    std::chrono::milliseconds extra_wait = 10ms; // extra 10ms wait time
```

Comment:
rather than poluting the code with std::chrono::milliseconds  in this way you should look at the definition of 'duration' in the events code and do something similar